src/crypto/crypto_hkdf.cc

```diff
@@ -87,6 +87,10 @@ Maybe<bool> HKDFTraits::AdditionalConfig(
       : info.ToByteSource();
 
   params->length = args[offset + 4].As<Uint32>()->Value();
+  // HKDF-Expand computes up to 255 HMAC blocks, each having as many bits as the
+  // output of the hash function. 255 is a hard limit because HKDF appends an
+  // 8-bit counter to each HMAC'd message, starting at 1.
+  constexpr size_t kMaxDigestMultiplier = 255;
```

Comment:
Nit: It might be better suited to make this a static, but just as suggestion.

Reply:
It was `static` before and I intentionally removed the keyword. Could you elaborate on why `static` would be beneficial here?